datacapturing/src/androidTest/java/de/cyface/datacapturing/backend/BackgroundServiceTest.java

```diff
@@ -1,6 +1,6 @@
 package de.cyface.datacapturing.backend;
 
-import static de.cyface.datacapturing.BundlesExtrasCodes.EVENT_HANDLING_STRATEGY_ID;
+import static de.cyface.datacapturing.BundlesExtrasCodes.*;
```

Comment:
My IDE decided since the last update that in the auto-format this is the way to go -.- Don't ask me why, I assumed you changes something in the eclipse auto-formatter file but on the other hand: I did not update the repository in which this file is located.